packages/vite/src/node/server/middlewares/transform.ts

```diff
@@ -23,6 +23,8 @@ import {
 } from '../../constants'
 import { isCSSRequest, isDirectCSSRequest } from '../../plugins/css'
 
+const NEW_DEPENDENCY_BUILD_TIMEOUT = 5000
```

Comment:
Let's add a comment explaining why we need this here.

Reply:
You got it!